genome_designer/genome_finish/contig_display_utils.py

```diff
@@ -1,17 +1,30 @@
 from collections import namedtuple
 
+import settings
+
+
 Junction = namedtuple('Junction',
         ['ref', 'ref_count', 'contig', 'contig_count'])
 
 
 def get_ref_jbrowse_link(contig, loc):
-        return (contig.parent_reference_genome.get_client_jbrowse_link() +
-                '&loc=' + str(loc))
+    sample_alignment = contig.experiment_sample_to_alignment
+    bam_dataset = sample_alignment.dataset_set.get(
+            type='BWA BAM')
```

Comment:
This is a workaround to using type=Dataset.TYPE.BWA_ALIGN, because this file is imported by main/models.py, so importing Dataset here causes a circular import error.  Thoughts?